Sources/Noora/Components/TextPrompt.swift

```diff
@@ -58,6 +70,17 @@ struct TextPrompt {
 
         content += "\(title != nil ? "\n" : "")\(titleOffset)\(prompt.formatted(theme: theme, terminal: terminal)) \(input)"
 
+        if !errors.isEmpty {
+            var errorMessage = "Validation errors:\n\(titleOffset)"
+
+            errorMessage += errors
+                .map { "- \($0.message)" }
```

Comment:
Very nitpicky, but what about using another bullet symbol? @asmitbm what do you think?
```suggestion
                .map { "· \($0.message)" }
```